src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -521,7 +523,7 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 	//#region macOS Touchbar
 
 	async newWindowTab(): Promise<void> {
-		this.windowsMainService.open({ context: OpenContext.API, cli: this.environmentMainService.args, forceNewTabbedWindow: true, forceEmpty: true });
+		this.windowsMainService.open({ context: OpenContext.API, cli: this.environmentMainService.args, forceNewTabbedWindow: true, forceEmpty: true, remoteAuthority: this.environmentMainService.args.remote || undefined });
```

Comment:
Why is the `|| undefined` needed, isn't that a optional property anyway?

Reply:
That's to avoid potential issues with the empty string